src/synthesizer/base_galaxy.py

```diff
@@ -105,6 +105,63 @@ def get_spectra_dust(self, emissionmodel):
 
         return sed
 
+    def transmission(self):
```

Comment:
Sorry @ConnorSF this all looks fine but you've added back in all these functions which were moved to be stand alone and better named functions in `sed.py`. Can you remove these and use those if you need them.

Reply:
These being `transmission`, `Al`, `A`, `Av`, and `A1500`

Reply:
No problem, will remove asap